meta/meta.go

```diff
@@ -296,6 +296,28 @@ func (m *Meta) CreateTable(dbID int64, tableInfo *model.TableInfo) error {
 	return m.txn.HSet(dbKey, tableKey, data)
 }
 
+// CreateView creates a view with tableInfo in database.
+func (m *Meta) CreateView(dbID int64, tableInfo *model.TableInfo) error {
```

Comment:
Can we reuse `CreateTable` function?

Reply:
I will add OrReplace support to this function, so I prefer not to reuse CreateTable function